### ISSUE TITLE
[Accessibility]MAS 4.2.1: (eclipse/button '') The Name property of a focusable element must not be null.

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3583]
The following accessibility issue needs investigation.

GitHubTags: #A11yTCS; #A11yMAS; #A11yAuto; #AI4D; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-NewFeature; #MAS4.2.1; #C+E-H; #JavatoolingforIntelliJ/Eclipse

**Note: Similar issue is found for all the buttons in the 'Storage Container' section**

**App: eclipse**

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0
**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**Element path:** button ''

**Issue Details:** The Name property of a focusable element must not be null. [Section 508 502.3.1](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology)

**How To Fix:** Provide a UI Automation Name property that concisely identifies the element.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}

![Screenshot (463)](https://user-images.githubusercontent.com/41109135/65408132-15e2d980-de02-11e9-8496-9322051d9b71.png)

**Comment by Flanker32** [comment]
@Tanushagottipati We have removed storage explorer in IDE toolkit, could you please help us verify and close the issue? Here is the [build](https://dev.azure.com/mseng/VSJava/_build/results?buildId=11079936)

**Comment by Tanushagottipati** [comment]
As mentioned storage accounts tab cannot be opened in the eclipse toolkit. As storage explorer has been deprecated, closing this bug and attached screenshot for reference.
Environment:
Eclipse IDE for Enterprise Java Developers.
Version: 2019-09 R (4.13.0)
Build id: 20190917-1200
Solution Installed: The Azure Toolkit for Eclipse 3.20.0
![Screenshot (709)](https://user-images.githubusercontent.com/41109135/70886657-fd99e980-2001-11ea-8ef8-b652fc4c7e07.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749687
